<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Лабараторна №2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="data_block">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Блок по роботі з даними</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Розмір вибірки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Вхідний файл:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Вихідний файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="input_file_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Імпортувати</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_file_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Вибрати</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sample_size">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>40</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_file">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>lab3.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="output_file">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>output.txt</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="model_block">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>441</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Налаштування моделі</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Степені полінома</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x1_pow">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x2_pow">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="x3_pow">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="grid_search">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>     Підібрати найкращий</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип полінома</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cheb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> Чебишева</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lagger">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> Лагерра</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ermit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>  Ерміта</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lejandr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Лежандра</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Розмірність даних</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>X3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_x1_size">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    <property name="placeholderText">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_x2_size">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    <property name="placeholderText">
      <string>2</string>
    </property>
    </widget>
    <widget class="QLineEdit" name="input_x3_size">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    <property name="placeholderText">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_y_size">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>290</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    <property name="placeholderText">
      <string>4</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Лямбда :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lambda_sep">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Окремо</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lambda_tog">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Разом</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>340</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Підрахунок b_q0:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="max_min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Максимум і мінімум</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="normed_y">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Нормований y</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Обчислити</string>
    </property>
   </widget>
   <widget class="QFrame" name="plot_block">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="plot">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Побудувати графік</string>
     </property>
    </widget>
    <widget class="QComboBox" name="var_choice">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>50</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="norm_plot">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>             Нормовані значення</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Візуалізація</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
